### ISSUE TITLE
Improve the logged metrics names

### ISSUE THREAD

**Issue by ibro45** [project-lighter/lighter#92]
## 🚀 Feature Request

When referencing the metric in, e.g., the `ModelCheckpointer`, specifying `Dice` is currently the correct way in this case https://github.com/ibro45/lighter-totalsegmentator-monai/pull/3#issuecomment-1891056053. However, it should have at least the corresponding mode prefix `e.g. val_Dice`, and, ideally, mimic the name as we log it in wandb and tensorboard `val/metrics/Dice`. However, the default way to reference validation loss is `val_loss`. Can we also achieve the same convention for it so that it is `val/loss` as well? Will make it much more consistent and less confusing.

Also, it's confusing how PL parses these names in the background. E.g. we had to do this here https://github.com/project-lighter/lighter/blob/e64694230e74f5dd6e7e29fbdc532346401219d6/lighter/system.py#L209 but the PL logged names (two lines below) doesn't really log it like that.

<img width="1083" alt="image" src="https://github.com/project-lighter/lighter/assets/18015788/719c64e1-fd81-47a7-acdb-311ebb289480">
